ui/apps/everest/src/pages/db-cluster-details/db-cluster-details.tsx

```diff
@@ -62,7 +66,25 @@ export const DbClusterDetails = () => {
           onBackClick={() => navigate('/databases')}
         />
         {/* At this point, loading is done and we either have the cluster or not */}
-        <DbActionButton dbCluster={dbCluster!} />
+        <Box
+          sx={{
+            display: 'flex',
+            alignItems: 'center',
+            flexWrap: 'wrap',
+            gap: 1,
+          }}
+        >
+          <DbActionButton dbCluster={dbCluster} />
+          {dbCluster.status?.status && (
+            <StatusField
+              dataTestId={dbClusterName}
+              status={dbCluster.status.status}
+              statusMap={DB_CLUSTER_STATUS_TO_BASE_STATUS}
+            >
+              {beautifyDbClusterStatus(dbCluster.status.status)}
+            </StatusField>
```

Comment:
```suggestion
           <StatusField
              dataTestId={dbClusterName}
              status={dbCluster.status?.status as DbClusterStatus}
              statusMap={DB_CLUSTER_STATUS_TO_BASE_STATUS}
          >
              {beautifyDbClusterStatus(dbCluster.status?.status as DbClusterStatus)}
          </StatusField>
```
посмотрела более внимательно, почему мы должны проверять status?.status если сам компонент статуса выполняет эту проверку и готов в случае чего подменить на unknown? +если статус не пришёл или не ясен это не значит что его нет, это значит что он unknown 
следоватльно компонент должен быть показан всегда

Reply:
предлагаю убрать условие проверки и оставить тот код который предложила в качестве альтернативы и поднять в upstream